_data/guides-main.yaml

```diff
@@ -327,8 +327,12 @@ categories:
         keywords: security
       - title: Using JWT RBAC
         url: /guides/security-jwt
-        description: This guide explains how your application can utilize SmallRye JWT to provide secured access to the JAX-RS endpoints.
+        description: This guide explains how your application can utilize SmallRye JWT to verify JWT tokens and provide secured access to the JAX-RS endpoints.
         keywords: security
+      - title: Build, Sign and Encrypt JSON Web Tokens (JWT)
```

Comment:
Weird, I don't see this entry in the /guides page

Reply:
@gastaldi Hmm, me neither, may be a `,` is a problem, let me try to remove it

Reply:
No that did not help, not sure what is the problem

Reply:
Hey @gastaldi, it is here: https://quarkusio-quarkusio-github-io-preview-pr-1194.surge.sh/version/main/guides/#security

I only updated `main`, https://github.com/quarkusio/quarkus/pull/21130 has not been backported, I added a backport label though, will update `latest` when backported